interfaces/src/db.ts

```diff
@@ -87,3 +93,55 @@ export interface Query {
   skip?: number;
   orderBy?: ReadonlyArray<Order>;
 }
+
+export interface ClientContext {
+  debugId: string;
```

Comment:
Should `debugId` be generated by the server in this case?

Reply:
We never do this with external APIs, we always generate our own debugId and return it to the client.